pkgs/applications/misc/bottles/default.nix

```diff
@@ -3,18 +3,18 @@
 , desktop-file-utils, gsettings-desktop-schemas, libnotify, libhandy
 , python3Packages, gettext
 , appstream-glib, gdk-pixbuf, glib, gobject-introspection, gspell, gtk3
-, steam-run, xdg-utils, pciutils, cabextract, wineWowPackages
+, steam-run, xdg-utils, pciutils, cabextract, wineWowPackages, webkitgtk, p7zip
 }:
 
 python3Packages.buildPythonApplication rec {
   pname = "bottles";
-  version = "2021.7.28-treviso-2";
+  version = "2021.12.14-treviso-4";
 
   src = fetchFromGitHub {
     owner = "bottlesdevs";
     repo = pname;
     rev = version;
-    sha256 = "0kvwcajm9izvkwfg7ir7bks39bpc665idwa8mc8d536ajyjriysn";
+    hash = "sha256-Lbqw/M37PUmf2af8B+G4IySifoTfBwMXw9UmeymWiNM=";
```

Comment:
Why this change ?

Reply:
`nix-build` indicated that's the "net style". At least it prints this kind of hash when there's a mismatch.

Reply:
> An alternative – and supposedly preferred – way of specifying hashes are so-called "SRI hashes". They're pretty simple, as the hash contains the algorithm used and always is in base64: 

https://nixos.wiki/wiki/Nix_Hash